.CMake/alg_support.cmake

```diff
@@ -384,6 +384,73 @@ endif()
 
 ##### OQS_COPY_FROM_UPSTREAM_FRAGMENT_ADD_ENABLE_BY_ALG_END
 
+option(OQS_ENABLE_SIG_STFL_XMSS "Enable XMSS algorithm family" ON)
```

Comment:
This should be documented in CONFIGURE.md

Reply:
Added in #1656 , let me know if it's good enough.